verify.js

```diff
@@ -35,9 +35,18 @@ const noDuplicates = Object.keys(db)
         console.error(`Failed to verify '${list}': '${numberOfItems - numberOfUniqueIdentifiers}' duplicate Identifier(s) found.`);
         return false;
       }
+
+      const uuidtest = db[list].map(({ uuid }, i) => {
+        if (! /^([A-F0-9]{4}|[A-F0-9]{8}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{12})$/.test(uuid)) {
```

Comment:
32-bit UUIDs are also allowed by spec, although not defined yet.

Reply:
@bencefr we can add them in this change or in a different one; either way is fine by me

Reply:
@bihanssen is looking at `ajv` if it is possible to validate the field by the schema. Maybe we'll move and extend the validation there.

Reply:
I'm ok with either solution. I haven't started on any implementation, just confirmed from the doc that ajv supports direct checking. I'll leave it to you to decide where to put it.

Reply:
I think we should merge this, and then continue in a separate change.